tests/httpd/config.ini.in

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+#***************************************************************************
+#                                  _   _ ____  _
+#  Project                     ___| | | |  _ \| |
+#                             / __| | | | |_) | |
+#                            | (__| |_| |  _ <| |___
+#                             \___|\___/|_| \_\_____|
+#
+# Copyright (C) 2008 - 2022, Daniel Stenberg, <daniel@haxx.se>, et al.
+#
+# This software is licensed as described in the file COPYING, which
+# you should have received as part of this distribution. The terms
+# are also available at https://curl.se/docs/copyright.html.
+#
+# You may opt to use, copy, modify, merge, publish, distribute and/or sell
+# copies of the Software, and permit persons to whom the Software is
+# furnished to do so, under the terms of the COPYING file.
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTY OF ANY
+# KIND, either express or implied.
```

Comment:
```suggestion
# KIND, either expressed or implied.
```

Reply:
"either express or implied." is the common way to phrase this. Also seen in numerous other files in curl and around the world.

Reply:
Oh, I see. It seemed to me like it should be past participle to match "implied" but I am apparently wrong

https://www.findlaw.com/consumer/consumer-transactions/what-are-express-and-implied-warranties.html